magento/magento2
# Order mail sender empty 16865

### Description (*)
Magento/Sales/Model/Order/Email/SenderBuilder.php
79+ $transport->setIdentityContainerStore($this->identityContainer->getStore());

### Fixed Issues (if relevant)

1. Transaction emails sender email is incorrect

### Manual testing scenarios (*)

1. Place and order and get order confirmation email
2. Check the sender email in order confirmation email

### Contribution checklist (*)
